utils.js

```diff
@@ -447,7 +447,7 @@ export const parsePkgJson = async (pkgJsonFile) => {
       const group = pkgIdentifier.scope || "";
       const purl = new PackageURL(
         "npm",
-        encodeForPurl(group),
+        group,
```

Comment:
I think this might break yarn project.

Reply:
Interesting, why do you think so? I've just tested with yarn projects locally and seems fine.

Reply:
Oh. Let me check. I remember adding it specifically for a project.